src/store/Store.js

```diff
@@ -46,6 +54,8 @@ export default class Store {
   _responses: StoreResponses;
   _state: any;
   _uid: string;
+  _devToolsExtension: DevToolsExtension;
```

Comment:
Should this be nullable? `_devToolsExtension: ?DevToolsExtension;`

Reply:
Turns out flow doesn't like this being nullable and having functions that may be called on it so gonna stick with it not being nullable. 

If there is a better way I'm happy to change that but I don't know flow enough to know if there is.